src/runtime/test/storage/crdt-collection-model-test.ts

```diff
@@ -6,43 +6,45 @@
 // http://polymer.github.io/PATENTS.txt
 
 import {assert} from '../../../platform/chai-web.js';
-import { CrdtCollectionModel } from '../../storage/crdt-collection-model.js';
+import { CrdtCollectionModel, ModelValue } from '../../storage/crdt-collection-model.js';
 
 describe('crdt-collection-model', () => {
   it('can add values', async () => {
+    const value: ModelValue = {id: 'value'};
     const model = new CrdtCollectionModel();
-    const effective = model.add('id', 'value', ['key1', 'key2']);
+    const effective = model.add('id', value, ['key1', 'key2']);
     assert.isTrue(effective);
     assert.equal(model.size, 1);
-    assert.equal(model.getValue('id'), 'value' as unknown);
+    assert.equal(model.getValue('id'), value);
     assert.sameMembers(model.getKeys('id'), ['key1', 'key2']);
   });
   it('can remove values', async () => {
     const model = new CrdtCollectionModel();
-    model.add('id', 'value', ['key1', 'key2']);
+    model.add('id', {id: 'value'}, ['key1', 'key2']);
```

Comment:
I think the intention is that if there's an id field in the value (which now there has to be) then it'll match the id passed into model.add. Probably best to follow that pattern in testing too, otherwise it's confusing for people looking at the code + tests.

Reply:
So this should be:

`model.add('id', {id: 'id'}, ['key1', 'key2']);`

right?

Reply:
It was probably going for something closer to:
```model.add('id', {id: 'id', value: 'value'}, ['key1', 'key2']);

Reply:
ack, will do.

Reply:
actually s/value:/rawData:/ and we're good.